h2o-hadoop/h2o-cdh5.10/build.gradle

```diff
@@ -4,3 +4,10 @@ ext {
 }
 
 apply from: '../driverjar.gradle'
+
+dependencies {
+  compile (project(":h2o-jetty-8")) {
+    exclude module: "servlet-api"
```

Comment:
Is it possible that there is also Jetty 6 in the hadoop build that would require this?